packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -154,6 +156,10 @@ export const Webchat = forwardRef((props, ref) => {
     menuIsOpened ? setMenuIsOpened(false) : setMenuIsOpened(true)
   }
 
+  const closeMenu = () => {
+    setMenuIsOpened(false)
+  }
+
   const sendInput = async input => {
     let isRegex = false
     let inputMessage = null
```

Comment:
this method is too long. Maybe we can split it using  a private method for text inputs?

Reply:
updated with the last PR